### ISSUE TITLE
 http://127.0.0.1:8100/inspector 无法看到iPhone屏幕投影，因为 WebDriverAgent 中的 /inspector 端点被移除了

### ISSUE THREAD

**Issue by lilis38** [AirtestProject/iOS-Tagent#277]
**Warning:**
![image](https://github.com/user-attachments/assets/1a79f9e9-dd4f-423a-866c-0788960db7da)

These issues are not tracked. Please create new issues in the main Appium
repository: https://github.com/appium/appium/issues/new

**Comment by nanoscopic** [comment]
Yeah Appium removed it because they became the maintainers of WebDriverAgent and don't want competition with the product they are pushing you to buy. :) Cool huh?
